packages/ardour-git/ardour-git.pacscript

```diff
@@ -11,8 +11,10 @@ build_depends="libgio2.0-cil-dev python3 libqm-dsp-dev libcairo2-dev libglib2.0-
 optdepends=("xjadeo: video monitoring"
 			"harvid: video timeline"
 			"linuxaudio-new-session-manager: for session management")
-gives="ladspa-host, lv2-host, vst-host, vst3-host"
+gives="ladspa-host, lv2-host, vst-host, vst3-host, ardour, ardour-data"
+breaks="ardour, ardour-data"
```

Comment:
```suggestion
breaks="ardour ardour-data"
```

Reply:
the commas must be added to match deb control file syntax